getDeployConfig.js

```diff
@@ -9,7 +9,12 @@ const projectRoot = require('app-root-path')
 const appPackage = require(path.join(projectRoot.path, 'package.json'))
 const commandLineArguments = require('yargs-parser')(process.argv.slice(2), { alias: { deployConfigFile: ['df'] } })
 const Logger = require('roosevelt-logger')
-const logger = new Logger()
+const params = {
+  params: {
+    disable: ['MOCHA_MODE'] // disable logging during Mocha tests
```

Comment:
we shouldn't call this mocha mode or even necessarily imply that it's for automated tests

it should be called something like "SILENT_MODE" so that it's context-agnostic